scheduler/scheduler-server/src/test/java/functionaltests/job/TestPreemptRestartKillTask.java

```diff
@@ -82,6 +86,16 @@
     private static URL jobDescriptor = TestPreemptRestartKillTask.class
             .getResource("/functionaltests/descriptors/Job_preempt_restart_kill.xml");
 
+    private static URL configFile = TestPreemptRestartKillTask.class
+            .getResource("/functionaltests/config/schedulerPropertiesNoRetry.ini");
```

Comment:
In case you use a custom scheduler configuration. It's better to extend the class SchedulerFunctionalTestWithCustomConfigAndRestart